lib/modules/platform/bitbucket-server/utils.spec.ts

```diff
@@ -268,7 +269,31 @@ describe('modules/platform/bitbucket-server/utils', () => {
             ),
           );
         });
+
+        it('works gitUrl:endpoint no basic auth', () => {
+          expect(
+            getRepoGitUrl(
+              'SOME/repo',
+              url.toString(),
+              'endpoint',
+              infoMock(url, 'SOME', 'repo'),
+              {},
+            ),
+          ).toBe(httpLink(url.toString(), 'SOME', 'repo'));
+        });
       });
     });
   });
+
+  describe('getExtraCloneOpts', () => {
+    it('should not configure bearer token', () => {
+      const res = getExtraCloneOpts({});
+      expect(res).toMatchSnapshot();
```

Comment:
no snapshots

https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md

Reply:
changed it